crates/typst/src/symbols/sym.rs

```diff
@@ -992,5 +992,5 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
     kelvin: 'K',
     Re: 'ℜ',
     Im: 'ℑ',
-    dotless: [i: '𝚤', j: '𝚥'],
+    dotless: [i: 'ı', j: 'ȷ'],
```

Comment:
This change makes this character act more in line with other Latin characters in math. (Some) Single characters in math are converted to their italic math variants, so the dotless i and j should as well. See the example below.

<table>
<tr>
<td> Code </td> <td> Render </td>
</tr>
<tr>
<td>

```typ
$ 𝑗 upright(𝑗) italic(𝑗) bb(𝑗) med
  𝚤 upright(𝚤) italic(𝚤) bb(𝚤) \
  j upright(j) italic(j) bb(j) med
  ı upright(ı) italic(ı) bb(ı) $
```

</td>
<td>

![test](https://github.com/user-attachments/assets/628ea46e-dc55-4e12-a8e5-dfa9dd2694c3)

</td>
</tr>
</table>